mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -588,6 +633,12 @@ const mapStateToProps = (state, ownProps) => {
     metricLists.push(Object.values(getLatestMetrics(runUuid, state)));
     paramLists.push(Object.values(getParams(runUuid, state)));
     const runTags = getRunTags(runUuid, state);
+    const visibleTags = Utils.getVisibleTagValues(runTags).map(([key, value]) => ({
+      key,
+      name: key,
+      value,
+    }));
+    tagLists.push(visibleTags);
```

Comment:
@CaioCavalcanti I pushed one commit to clean up the code here.

Reply:
Nice, good one